dbt_server/state.py

```diff
@@ -232,11 +277,22 @@ def compile_query(self, query):
 
     @tracer.wrap
     def execute_query(self, query):
+        """Returns a result dict of executing `query` according to current
+        state."""
         return dbt_service.execute_sql(self.manifest, self.root_path, query)
 
     @tracer.wrap
-    def execute_async_command(self, task_id, command, db, callback_url) -> None:
-        return dbt_service.execute_async_command(
+    def execute_async_command(
+        self, task_id: str, command: List[str], db: Any, callback_url: str
```

Comment:
```suggestion
        self, task_id: str, command: List[str], db: Any, callback_url:  Optional[str] = None
```
This was our oversight, this should not be mandatory